x-pack/plugins/enterprise_search/public/applications/app_search/components/create_engine/create_engine.tsx

```diff
@@ -43,6 +43,7 @@ export const CreateEngine: React.FC = () => {
   const { name, rawName, language } = useValues(CreateEngineLogic);
   const { setLanguage, setRawName, submitEngine } = useActions(CreateEngineLogic);
 
+  // TODO these need to come from AppLogic and/or the Enterprise Search server
```

Comment:
If you're following along commit-by-commit I remove this in [`207f8b`](https://github.com/elastic/kibana/pull/89816/commits/207f8b66ab28de417de5db43f4baee43ae632454#diff-529e95b2a96ac2a4737364f8304d2c07b6095cb2378c51fb63a329faf8d5b86aL46-L51)